### DISCUSSION TITLE
ChartBoy, a PHP library to create charts.css charts easily

### DISCUSSION THREAD

**Discussion by wolfgangs-code** [ChartsCSS/charts.css#58] [Show and tell]
Hi! I just made and released **ChartBoy**, a Composer package to make charts.css charts easily with PHP. It's trying to go for getting every feature, but right now it's missing only a few (multiple datasets as of 0.9.0).

[The full documentation is here](https://wolfgang-degroot.github.io/ChartBoy/), and getting a chart ready goes like this:
```php
$chart = new ChartBoy\ChartBoy($dataArray, "column");
$chart->changeSetting("show-labels", true);
$chart->changeSetting("show-*-secondary-axes", 4);
$chart->renderChart("chart-html-id");
```
_It's up to you to make a container before rendering the chart, though._
My preference is to do `<div><?=$chart->renderChart();?></div>`

---------------------

ChartBoy's major/minor versions match charts.css to signify which version is compatible, while the patch number its specific to ChartBoy. e.g. Chartboy _0.9.13_ would signify compatibility with charts.css _0.9.x_

😊

**Comment by ramiy** [comment]
Added to https://github.com/ChartsCSS/awesome
